uievents/click/dblclick_event_mouse.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
```

Comment:
Does this now show a better error for Safari?

Reply:
Well, actually it doesn't, because it what happens is that two "click" events are fired but no "dblclick". So it's still a timeout.

Some of the simplification and the renaming still seems worthwhile, though.

Reply:
I've updated the commit message not mention Safari. Still a timeout.